classification/templates/classification/condition_match_test.html

```diff
@@ -88,6 +88,7 @@ <h4>Test Single Term</h4>
                     {% endlabelled %}
                     {% if suggestion.terms %}
                         {% labelled label="Can auto-assign (without confirmation)" %}{{ is_auto_assignable|boolean }}{% endlabelled %}
+                        {% labelled label="Multi Condition Term" %}{{ suggestion.condition_multi_operation }}{% endlabelled %}
```

Comment:
Would be great to show

suggestion.condition_multi_operation.label

and to only show the field at all if `suggestion.terms|len > 1`